generated/react-native-flexible-sync/realm.json

```diff
@@ -1,4 +0,0 @@
-{
-  "appId": "todo-sync-zpqwm",
```

Comment:
Needed fix: The generated files should have a `realm.json`.

Reply:
I don't think this is technically required since the "create a new template app" process that end users go through adds a `realm.json`. We ignore `realm.json` for web which is why it's ignored here too. That said, if it makes it easier to develop by including it here we can add it back.

Reply:
oh i see, it gave me an error when just cloning the source directly, but if it's created through that process its fine